jaeger_client/tracer.py

```diff
@@ -80,16 +79,19 @@ def __init__(
             self.codecs.update(extra_codecs)
         self.tags = {
             constants.JAEGER_VERSION_TAG_KEY: constants.JAEGER_CLIENT_VERSION,
-            constants.JAEGER_IP_TAG_KEY: self.ip_address,
         }
         if tags:
             self.tags.update(tags)
-        # noinspection PyBroadException
-        try:
-            hostname = socket.gethostname()
-            self.tags[constants.JAEGER_HOSTNAME_TAG_KEY] = hostname
-        except:
-            logger.exception('Unable to determine host name')
+
+        if self.tags.get(constants.JAEGER_IP_TAG_KEY) is None:
+            self.tags[constants.JAEGER_IP_TAG_KEY] = ipv4_to_int(local_ip())
```

Comment:
I suggest we remove `ipv4_to_int` portion while at it - it made sense in Zipkin format which had IP as an int field, but Jaeger converts all tags to strings, so this is no longer useful (in fact we have enricher in the backend that does the reverse conversion). Also, with string we can support ipv6.